src/blissful_tuner/fp8_optimization.py

```diff
@@ -368,3 +370,106 @@ def new_forward(self, x):
     logger.info(f"Quantization done in {q_dtype}")
     logger.info(f"Number of monkey-patched Linear layers: {patched_count}")
     return model
+
+
+def load_safetensors_with_fp8_optimization(
```

Comment:
Good call to have this into your module.

I thought about doing the same for a number of customizations and simply overwriting the local function or adding a proxy decorator to enable dynamically choosing the blissful or musubi implementation.

I ultimately decided against it as the api's are simply not stable so silent failures would occur without unit tests to catch them. Still going to run into that situation here, but it's less 🪄magic🪄.